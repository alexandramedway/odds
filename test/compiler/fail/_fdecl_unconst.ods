do f = (x) ->
    do y = 1 + 2
    return x
