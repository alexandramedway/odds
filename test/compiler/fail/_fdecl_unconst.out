Analyzer Exception: Invalid declaration of function 'f' with unconstrained return value
